### ISSUE TITLE
Does this project alive????

### ISSUE THREAD

**Issue by deios0** [Fritak/messenger-platform#25]
I've asked for changes more than a month ago.
Please, if you will not be able to do the changes, tell me.

**Comment by achimha** [comment]
It seems dead unfortunately. Hope somebody can fork and enhance it. I haven't found any other PHP implementation that actually works.

**Comment by deios0** [comment]
Yes, I think so too, actually, it's weird, why there is still no any good PHP implementation of Facebook Messenger API?

**Comment by darrensapalo** [comment]
@deios0 Indeed, it is weird. I've been trying to look for live ones. Try checking this one out: 

https://github.com/davidpiesse/facebook_messenger_php

**Comment by achimha** [comment]
I switched to this one:

https://github.com/pimax/fb-messenger-php

It's a very thin layer, you end up dealing with the JSON directly as you get it from Facebook which is OK.

**Comment by deios0** [comment]
Yes, I switched to pimax too, several weeks ago, looks good for me.
